distributed_index_tests/tests/dist_index_sanity_10k_docs.py

```diff
@@ -1,73 +1,41 @@
 import time
-from threading import Thread
-import pytest
-
-from prov.reset_sync_gateway import reset_sync_gateway
-
-from lib.syncgateway import SyncGateway
-from lib.user import User
-
-from data.data import Doc
+from lib.admin import Admin
 
 from cluster_setup import cluster
 
-def doc_spliter(docs, batch_num=100):
-    for i in xrange(0, len(docs), batch_num):
-        yield docs[i:i+batch_num]
-
-
-def issue_requests_for_docs(sgs, docs, user):
-    count = 0
-    for doc in docs:
-        sg_index = count % len(sgs)
-        doc_name, doc_body = doc.name_and_body()
-        r = sgs[sg_index].db.add_user_document(doc_name, doc_body, user)
-        assert(r.status_code == 201)
-        print(r.status_code)
-
 
 def test_1(cluster):
```

Comment:
We need a scope for this pytest fixture: Module, test etc. 
Also need a teardown function which gets executed as soon as this test/object gets out of scope.